### DISCUSSION TITLE
FlowNext Support

### DISCUSSION THREAD

**Discussion by sidilarsen** [LibreHardwareMonitor/LibreHardwareMonitor#492] [Q&A]
Hi all.

Do you support the aquacomputer Flownext Sensor ?
https://shop.aquacomputer.de/product_info.php?products_id=3953&language=en
Hardware info seems to support it 
If not do you plan to support it ?

Best regards 
Steve

**Comment by EyoSama** [comment]
I have the exact same question. It would be awesome if this sensor were supported, as it has a slew of different incorporated sensor information. I use TTController from MoshiMoshi, which uses LHM. TTController allows controlling the Thermaltake Fans and LEDs, and does an incredible job at it (a much better job than the actual Thermaltake software, which is unusable). The combination of TTController and LHM, is simply amazing and very impressive. Kudos to everyone involved!

But, one thing that would make it even more functional, would be the ability to be able to control the Fans based on the Water Temperature of a Liquid Cooling loop, since it makes much more sense to have the Fans react to the water temperature and not the individual GPU and CPU Temps. If LHM would support the FlowNext and/or also supported the WATER_IN and WATER_OUT sensors on most motherboards, then that would be achievable.

Would this be possible?
